crypto/fipsmodule/cpucap/cpu_ppc64le.c

```diff
@@ -24,11 +24,56 @@
 #define PPC_FEATURE2_HAS_VCRYPTO 0x02000000
 #endif
 
+static void handle_cpu_env(unsigned long *out, const char *in) {
```

Comment:
Could put this function implementation in an internal header file instead of duplicating?

Reply:
The existing implementations are very similar but not quite compatible with each other:
* Intel: https://github.com/aws/aws-lc/blob/33637ff1765823bef584c7959a5ef84e6f7bbc9f/crypto/fipsmodule/cpucap/cpu_intel.c#L131
* ARM64: https://github.com/aws/aws-lc/blob/33637ff1765823bef584c7959a5ef84e6f7bbc9f/crypto/fipsmodule/cpucap/cpu_aarch64.c#L8

This implementation does appear to be compatible with "Intel", except the "Intel" implementation assumes little-endian while this one should also work for big-endian.

Reply:
Do we want to handle in intel and ppc the case of trying to enable a non-existent capability as in arm https://github.com/aws/aws-lc/blob/33637ff1765823bef584c7959a5ef84e6f7bbc9f/crypto/fipsmodule/cpucap/cpu_aarch64.c#L27

Reply:
> Do we want to handle in intel and ppc the case of trying to enable a non-existent capability as in arm.

This would invalidate the "OR" use-case [here](https://github.com/aws/aws-lc/blob/33637ff1765823bef584c7959a5ef84e6f7bbc9f/crypto/fipsmodule/cpucap/cpu_intel.c#L152) unless it's likewise safe to ignore the existing `out` values when they are `0`.

Reply:
Added rejection logic in latest commit.